frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -1212,9 +1212,11 @@ public void doView(RenderRequest request, RenderResponse response) throws IOExce
     }
 
     private void prepareStandardView(RenderRequest request) throws IOException {
+        Set<String> projectGroups = new HashSet<String>();
         User user = UserCacheHolder.getUserFromRequest(request);
-        List<Organization> organizations = UserUtils.getOrganizations(request);
-        request.setAttribute(PortalConstants.ORGANIZATIONS, organizations);
+        List<Project> projects = getFilteredProjectList(request);
+        projectGroups = getProjectGroups(projects);
```

Comment:
No need to for line 1215 and can be written as 
`Set<String> projectGroups = getProjectGroups(projects);`
because `getProjectGroups()` handles empty set of groups

Reply:
Fixed, Please check.